charts/tenantfilesystem/templates/xrd.yaml

```diff
@@ -0,0 +1,113 @@
+# Crossplane Composite Resource Definition (XRD) describing
+# the type and schema of the corresponding Composite Resource.
+# See https://negz.github.io/crossplane.github.io/docs/v1.4/reference/composition.html#compositeresourcedefinitions for details
+
+apiVersion: apiextensions.crossplane.io/v1
+kind: CompositeResourceDefinition
+metadata:
+  name: xtenantfilesystems.wadtfy.bmwgroup.net
```

Comment:
We should add an additional property `deletionPolicy` which can be set to either `delete` or `orphaned`. This might be necessary for future upgrades/migrations of the component where we want to avoid components from being destroyed by crossplane.